Tests/Command/DumpSchemaCommandTest.php

```diff
@@ -1,17 +1,18 @@
 <?php
 
-
 namespace TheCodingMachine\Graphqlite\Bundle\Tests\Command;
 
-
 use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
 use TheCodingMachine\Graphqlite\Bundle\Tests\GraphqliteTestingKernel;
-use Symfony\Bundle\FrameworkBundle\Console\Application;
 
 class DumpSchemaCommandTest extends TestCase
 {
-    public function testExecute()
+    /**
+     * @test
+     */
+    public function it_executes_successfully(): void
```

Comment:
Why? That looks like phpspec style, not what is generally used in phpunit

Reply:
This is a recommended practice to name your tests to make them descriptive and indicative of what tests are doing.
I don't think this is something specific to _phpspec_.

Reply:
Okay, let's say it's a matter of taste. This project, along with the graphqlite package doesn't use this syntax, and I don't see any point in changing it, especially if you change only a single test method.

Reply:
Ok reverted this too.